runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/Engine.java

```diff
@@ -102,7 +106,8 @@ public final class Engine implements AutoCloseable
         Collection<MetricGroup> metricGroups,
         Collection<Vault> vaults,
         ErrorHandler errorHandler,
-        Collection<EngineAffinity> affinities)
+        Collection<EngineAffinity> affinities,
+        boolean readOnly)
```

Comment:
We have precedent in the codebase for spelling this `readonly` instead.
Please rename all occurrences of `readOnly` to `readonly`.

Reply:
sure, done.